### DISCUSSION TITLE
Alchemical Symbols

### DISCUSSION THREAD

**Discussion by trzyem** [psb1558/Junicode-font#299] [General]
Thanks for the hard work and the great font.
Are there any plans to add astronomical and/or alchemical symbols?

https://en.m.wikipedia.org/wiki/Alchemical_Symbols_(Unicode_block)

**Comment by psb1558** [comment]
I would say not in the near term. With symbols like these, which don't have to match the style of the surrounding text, it is just as good a solution to shift into another font to access them. On my system, they are available in the Quivira and Apple Symbols fonts, and I understand they are also in a font called Symbola. There are probably others as well.

**Reply by trzyem** [reply]
Thank you!
